File: src/main/java/it/contrader/dto/MapDTO.java
package it.contrader.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class MapDTO {
	
	private Long idMap;
	
	private Long idFolder;
	
	private String mapName;
	
	private String elixirmap;
	

}

File: src/dto/mapdto.ts
export class MapDTO {

id? : number;
idfolder : number;
mapName : string;
elixirMap : string;

constructor(id: number, idfolder: number, mapName: string, elixirMap: string) {
    this.id = id;
    this.idfolder = idfolder;
    this.mapName = mapName;
    this.elixirMap = elixirMap;
}

  

}

